Last-Update: 2014-12-13
Forwarded: no
Author: Dmitry Borisyuk" <q1werty@i.com.ua>
Bug-Debian: http://bugs.debian.org/772080
Description: prevent hangs on extraction from encrypted .ZIP archives, suppress password prompt with dummy password.

--- a/src/vfs/extfs/helpers/uzip.in
+++ b/src/vfs/extfs/helpers/uzip.in
@@ -32,7 +32,7 @@
 # Command used to delete a file from the archive
 my $cmd_delete = "$app_zip -d";
 # Command used to extract a file to standard out
-my $cmd_extract = "$app_unzip -p";
+my $cmd_extract = "$app_unzip -P '' -p";
 
 # -rw-r--r--  2.2 unx     2891 tx     1435 defN 20000330.211927 ./edit.html
 # (perm) (?) (?) (size) (?) (zippedsize) (method) (yyyy)(mm)(dd).(HH)(MM)(SS) (fname)
